.github/workflows/build.yml

```diff
@@ -46,6 +46,13 @@ jobs:
           pip install coveralls
           coveralls
 
+      - name: Publish to PyPI
+        if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags') && matrix.python-version == '3.7'
+        uses: pypa/gh-action-pypi-publish@master
+        with:
+          user: __token__
+          password: ${{ secrets.pypi_password }}
```

Comment:
Can we make this secrets.pypi_token?

Reply:
Yes that would be better. Only somehow it has broken it. Now I'm doubting that I got organisation secrets working. Continuing to investigate ...

Reply:
Ah - the magic is don't but a carriage return at the end of your token